app/src/renderer/components/staking/LiDelegate.vue

```diff
@@ -1,9 +1,13 @@
 <template lang='pug'>
 .li-delegate(:class='styles'): .li-delegate__values
-  .li-delegate__value.name
-    span
-      router-link(v-if="config.devMode" :to="{ name: 'delegate', params: { delegate: delegate.id }}")  {{ ' ' + delegate.moniker }}
-      a(v-else) {{ ' ' + delegate.moniker }}
+  .li-delegate__value.name: span
+    router-link(v-if="config.devMode"
+      :to="{ name: 'delegate-index', params: { delegate: delegate.id }}")
+      .li-delegate__icon(v-if="delegate.isValidator")
+        img(src="~assets/images/cosmos-validator.png")
```

Comment:
i don't know how i feel about the icon in the table. maybe on validator profile pages? maybe both? not sure...